docs/source/references/config_files/project/base_yml.rst

```diff
@@ -185,8 +209,48 @@ these settings have to be their default values as shown in the
 
       :port:
         | The port on which the service can be reached
-        | ``string``
+        | ``integer``
     
+Integrations
+++++++++++++++++
+
+Useful applications that can connect to Django, like sentry, cms, can be
```

Comment:
- Capitalize "sentry" and link it to https://sentry.io/welcome/.
- Remove cms from here.